python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -681,6 +681,24 @@ def __matmul__(self, other):
     def __rmatmul__(self, other):
         return _fast_slow_function_call(operator.matmul, other, self)[0]
 
+    def __isub__(self, other):
+        return _fast_slow_function_call(
+            lambda x, y: x.__isub__(y)
+            if hasattr(x, "__isub__")
+            else NotImplemented,
+            self,
+            other,
+        )[0]
+
+    def __idd__(self, other):
```

Comment:
There's a typo in this method name. Is this being tested?
```suggestion
    def __iadd__(self, other):
```